playbook/app/pb_kits/playbook/pb_timeline/_timeline.scss

```diff
@@ -163,6 +166,17 @@ $height_from_top: $icon_height/2 - $connector_width/2;
         @include flex_wrapper(column);
         align-items: flex-start;
         align-self: auto;
+        [class=pb_timeline_item_step] {
+            position: relative;
+        }
+        .step_children_container {
+            position: absolute;
+            left: 50%;
+            transform: translate(-50%, 0);
```

Comment:
Is there a reason to not use flex properties here? I usually avoid position:absolute like the plague, if possible. It can create problems sometimes, especially in nitro. 

Could you give this one a go with either a <Flex> kit below instead of a div for the step_children_container OR just use some flex css properties here.